README.md

```diff
@@ -604,6 +604,7 @@ Table of Contents
   * [scrapinghub.com](http://scrapinghub.com) — Data scraping with visual interface and plugins. Free plan includes unlimited scraping on a shared server
   * [context.io](https://context.io) — Create simple email webhooks and code against a free, RESTful, imap API to leverage email data
   * [cloudrail.com](https://cloudrail.com) — API integration solution. Unified interfaces for multiple web APIs, API change management
+  * [init.ai](https://www.init.ai/platform/automate) — Automate customer conversations, build intelligent conversational experiences.
```

Comment:
this states "free while we're in beta" with no indication that they intend to have a free tier for devs, so I'm afraid i cant accept it till thats clearer